resources/lang/misc.py

```diff
@@ -0,0 +1,43 @@
+from mcresources import ResourceManager
+
+from constants import *
+
+def generate(rm: ResourceManager):
+    # Item Groups
+    rm.lang(('itemGroup.tfc.rock.blocks', 'TFC Rock Blocks'))
```

Comment:
A couple things about `rm.lang`:
 - The tuple here isn't necessary, sequential arguments work fine. But more so,
 - In cases like this where there are many different lines, it's probably simpler to use the dictionary invocation:
```python
rm.lang({
    'itemGroup.tfc.rock.blocks': 'TFC Rock Blocks',
    'itemGroup.tfc.metals': 'TFC Metals',
...
})
```

This also has benifits for grouping similar calls:
```python
rm.lang(dict((
    'tfc.enum.tier.tier_%s' % tier,
    'Tier %s' % tier.upper()
) for tier in ('0', 'i', 'ii', 'iii', 'iv', 'v', 'vi'))
```